src/transaction/log_manager.c

```diff
@@ -14684,6 +14733,457 @@ cdc_is_filtered_user (char *user)
+
+	  if (cdc_get_recdes (thread_p, NULL, NULL, &redo_lsa, &redo_recdes, true) != NO_ERROR)
+	    {
+	      goto exit;
+	    }
+
+	  log_info_entry = (CDC_LOGINFO_ENTRY *) db_private_alloc (thread_p, sizeof (CDC_LOGINFO_ENTRY));
+	  if (log_info_entry == NULL)
+	    {
+	      error = ER_OUT_OF_VIRTUAL_MEMORY;
+	      goto exit;
+	    }
+
+	  error =
+	    cdc_make_dml_loginfo (thread_p, trid, tran_user,
+				  rec_type == LOG_SUPPLEMENT_INSERT ? CDC_INSERT : CDC_TRIGGER_INSERT, classoid, NULL,
+				  &redo_recdes, log_info_entry, true);
+
+	  if (error == NO_ERROR)
+	    {
+              // *INDENT-OFF*
+              context->loginfo_queue.push (log_info_entry);
+              // *INDENT-ON*
+	      context->queue_size += log_info_entry->length;
+	      num_loginfo++;
+	      log_info_entry = NULL;
+	    }
+	  else
+	    {
+	      goto exit;
+	    }
+
+	  break;
+	case LOG_SUPPLEMENT_UPDATE:
+	case LOG_SUPPLEMENT_TRIGGER_UPDATE:
+	  memcpy (&classoid, supplement_data, sizeof (OID));
+	  /* classoid filter */
+	  if (!flashback_class_filter (context, classoid))
+	    {
+	      goto end;
+	    }
+
+	  memcpy (&undo_lsa, supplement_data + sizeof (OID), sizeof (LOG_LSA));
+	  memcpy (&redo_lsa, supplement_data + sizeof (OID) + sizeof (LOG_LSA), sizeof (LOG_LSA));
+
+	  if ((error = cdc_get_recdes (thread_p, &undo_lsa, &undo_recdes, &redo_lsa, &redo_recdes, true)) != NO_ERROR)
+	    {
+	      goto exit;
+	    }
+
+	  log_info_entry = (CDC_LOGINFO_ENTRY *) db_private_alloc (thread_p, sizeof (CDC_LOGINFO_ENTRY));
+	  if (log_info_entry == NULL)
+	    {
+	      error = ER_OUT_OF_VIRTUAL_MEMORY;
+	      goto exit;
+	    }
+
+	  if (undo_recdes.type == REC_ASSIGN_ADDRESS)
+	    {
+	      /* This occurs when series of logs are appended like
+	       * INSERT record for reserve OID (REC_ASSIGN_ADDRESS) then UPDATE to some record.
+	       * And this is a sequence for INSERT a record with OID reservation.
+	       * undo record with REC_ASSIGN_ADDRESS type has no undo image to extract, so this will be treated as INSERT
+	       * CUBRID engine used to do INSERT a record like this way,
+	       * for instance CREATE a class or INSERT a record by trigger execution */
+
+	      assert (rec_type == LOG_SUPPLEMENT_TRIGGER_UPDATE);
+
+	      error =
+		cdc_make_dml_loginfo (thread_p, trid, tran_user, CDC_TRIGGER_INSERT, classoid, NULL, &redo_recdes,
+				      log_info_entry, true);
+	    }
+	  else
+	    {
+	      error =
+		cdc_make_dml_loginfo (thread_p, trid, tran_user,
+				      rec_type == LOG_SUPPLEMENT_UPDATE ? CDC_UPDATE : CDC_TRIGGER_UPDATE, classoid,
+				      &undo_recdes, &redo_recdes, log_info_entry, true);
+	    }
+
+	  if (error == NO_ERROR)
+	    {
+              // *INDENT-OFF*
+              context->loginfo_queue.push (log_info_entry);
+              // *INDENT-ON*
+	      context->queue_size += log_info_entry->length;
+	      num_loginfo++;
+	      log_info_entry = NULL;
+	    }
+	  else
+	    {
+	      goto exit;
+	    }
```

Comment:
like others do. Just suggestion.
```suggestion

	  if (error != NO_ERROR)
	    {
	      goto exit;
	    }
	   // *INDENT-OFF*
              context->loginfo_queue.push (log_info_entry);
              // *INDENT-ON*
	      context->queue_size += log_info_entry->length;
	      num_loginfo++;
	      log_info_entry = NULL;
```

Reply:
OK, thank you!